docs/source/examples/ggi_summer_school_solved.md

```diff
@@ -0,0 +1,313 @@
+---
+hidden: true
+
```

Comment:
Not sure why this is not working to fix the `toctree` error [here.](https://github.com/unitaryfund/mitiq/actions/runs/4238482095/jobs/7365600462#step:6:252) 

```
mitiq/mitiq/docs/source/examples/ggi_summer_school_solved.md: WARNING: document isn't included in any toctree
```

Reply:
should have used `orphan`